src/containers/NonBitGoRecovery/Coin/Coin.tsx

```diff
@@ -3,6 +3,40 @@ import { useParams } from 'react-router-dom';
 import { BitcoinForm } from './BitcoinForm';
 import { EthereumForm } from './EthereumForm';
 import { RippleForm } from './RippleForm';
+import { BitcoinCashForm } from './BitcoinCashForm';
+import { LitecoinForm } from './LitecoinForm';
+import { BitcoinABCForm } from './BitcoinABCForm';
+import { BitcoinSVForm } from './BitcoinSVForm';
+import { TronForm } from './TronForm';
+import { ERC20Form } from './ERC20TokenForm';
+
+import { Chain, Hardfork } from '@ethereumjs/common';
```

Comment:
you're not using these, remove them, and you can't import Node.js APIs in the browser runtime